.github/workflows/build.yml

```diff
@@ -0,0 +1,272 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
This could also work on PRs, to validate them before merging?

Reply:
done